tests/autofix_files/trio91x_autofix.py

```diff
@@ -39,10 +38,10 @@ async def foo_yield():  # TRIO911: 0, "exit", Statement("yield", lineno+2)
 
 
 async def foo_if():
-    if ...:
+    if foo():
```

Comment:
since `foo` is async, should we `if bar()` in these places?

Reply:
huh, I'd expect that to trigger [mypy's unused-coroutine](https://mypy.readthedocs.io/en/latest/error_code_list.html#check-that-coroutine-return-value-is-used-unused-coroutine) but I guess the return value is technically used.
Reported it at https://github.com/python/mypy/issues/16921#issue-2138489669

For the test file it of course doesn't really matter, unless mypy starts treating `if foo():` as `if True:` :grin:, but yeah no reason not to.